homeassistant/components/panasonic_viera/__init__.py

```diff
@@ -119,8 +119,8 @@ async def async_unload_entry(hass, config_entry):
     return unload_ok
 
 
-class Remote:
```

Comment:
Why the need to rename this class ?

Reply:
I found it confusing with the other Remote entity class in the remote.py. 
I thought it was better to rename it to device for a clear distinction.

Reply:
I think we should keep refactor out of scope, in order to be able to pull it into beta's or patch version.

Reply:
No problem. I'm reverting this change